packages/shared/providers.yaml

```diff
@@ -4469,6 +4470,8 @@ pennylane:
         grant_type: authorization_code
     refresh_params:
         grant_type: refresh_token
+    authorization_url_skip_encode:
+        - scopes
```

Comment:
I would personally prefer to bundle it with the `authorization_url` or `token_url` attributes. Something like:
```
authorization_url: 
   url: https://app.pennylane.com/oauth/authorize
   skip_encode: scopes
```
but it would require changing the type of an existing attributes 😓

Reply:
I agree. I'm not sure, but to me, this might be the best time to clean up and bundle all related fields together for both `authorization_url` and `token_url`. Currently, we have loosely grouped fields for `authorization_url`, such as:
```
authorization_url_replacements
authorization_url_fragment
```

Bundling them together would make the `providers.yaml` file cleaner. @TBonnin what do you think?

Reply:
grouping them together would make sense. Not in this PR for course. Maybe adding an 
```
authorization:
    url: ...
    url_replacements:...
    ...
```
and same thing with token. We would need to ensure backward compatibilities though. 😓